src/Mod/Fem/App/FemPostBranchFilter.h

```diff
@@ -0,0 +1,78 @@
+/***************************************************************************
+ *   Copyright (c) 2024 Stefan Tröger <stefantroeger@gmx.net>              *
+ *                                                                         *
+ *   This file is part of the FreeCAD CAx development system.              *
+ *                                                                         *
+ *   This library is free software; you can redistribute it and/or         *
+ *   modify it under the terms of the GNU Library General Public           *
+ *   License as published by the Free Software Foundation; either          *
+ *   version 2 of the License, or (at your option) any later version.      *
+ *                                                                         *
+ *   This library  is distributed in the hope that it will be useful,      *
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+ *   GNU Library General Public License for more details.                  *
+ *                                                                         *
+ *   You should have received a copy of the GNU Library General Public     *
+ *   License along with this library; see the file COPYING.LIB. If not,    *
+ *   write to the Free Software Foundation, Inc., 59 Temple Place,         *
+ *   Suite 330, Boston, MA  02111-1307, USA                                *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef Fem_FemPostBranchFilter_H
+#define Fem_FemPostBranchFilter_H
+
+
+#include "FemPostFilter.h"
+#include "FemPostGroupExtension.h"
+
+#include <vtkSmartPointer.h>
+#include <vtkAppendFilter.h>
+#include <vtkPassThrough.h>
+
+
+namespace Fem
+{
+
+class FemExport FemPostBranchFilter: public Fem::FemPostFilter, public FemPostGroupExtension
+{
+    PROPERTY_HEADER_WITH_EXTENSIONS(Fem::FemPostBranchFilter);
+
+public:
+    /// Constructor
+    FemPostBranchFilter();
+    ~FemPostBranchFilter() override;
+
+    App::PropertyEnumeration Output;
+
+
+    short mustExecute() const override;
+    PyObject* getPyObject() override;
+
+    constexpr const char* getViewProviderName() const override
```

Comment:
I know I wrote this should be constexpr. However it cannot be... due to some inheritance? Anyways, my fault!